CrowdFundingApplication/wwwroot/css/LogAndAutho.css

```diff
@@ -0,0 +1,52 @@
+﻿﻿@import url('https://fonts.googleapis.com/css2?family=Signika+Negative:wght@400;500;600&display=swap');
+
+body {
+    font-family: "Signika", sans-serif;
+    font-weight: 500;
+    letter-spacing: 1px;
+    background-image: url("../Images/img1.jpg");
+    align-items: center;
+    justify-content: center;
+    background-size: cover;
+    background-position: center;
+}
+
+.overlay {
+    position: relative;
+    z-index: 2;
+}
+
+    .overlay::after {
+        content: "";
+        width: 100%;
+        height: 100%;
+        position: absolute;
+        top: 0;
+        left: 0;
+        background-color: rgba(36,44,48,0.4);
+        z-index: -1;
+    }
+
+hr {
+    background-color: gray;
+    height: 1px;
+    margin: 18px auto;
+    opacity: 15%;
+}
+
+.btn-primary {
+    background-color: #a30846;
+    border-color: #a30846;
+    border-radius: 8px;
+}
+
+    .btn-primary:active {
+        background-color: #a30846 !important;
```

Comment:
It is better to override this for all buttons in a common style for the entire application.